app/tests/e2e/test_connections.py

```diff
@@ -109,32 +108,13 @@ async def test_delete_connection(
 async def test_bob_and_alice_connect(
     bob_member_client: AsyncClient,
     alice_member_client: AsyncClient,
+    bob_and_alice_connection: BobAliceConnect,
```

Comment:
I'm not sure we should use a fixture here. What are we trying to test? If we're testing the connection process we should write out the connection process I think.

Reply:
fair point